Tests/ApolloTests/WebSocket/WebSocketTests.swift

```diff
@@ -62,7 +62,7 @@ class WebSocketTests: XCTestCase {
   func testLocalSingleSubscription() throws {
     let expectation = self.expectation(description: "Single subscription")
     
-    client.subscribe(
+    _ = client.subscribe(
```

Comment:
We should probably be cancelling the subscriptions at the end of these tests as well!

Reply:
I added them but I'm less convinced of the value of doing it in these tests since they all exit and get torn down afterwards. There might be some negligible benefit for CI resources.